services/slam/builtin/cartographer_int_test.go

```diff
@@ -156,8 +176,136 @@ func TestCartographerIntegration(t *testing.T) {
 
 	// Close out slam service
 	test.That(t, utils.TryClose(context.Background(), svc), test.ShouldBeNil)
+
+	// Don't clear out the directory, since we will re-use the maps for the next run
+	closeOutSLAMService(t, "")
+
+	// added sleep to ensure cartographer stops
+	time.Sleep(time.Millisecond * cartoSleepMS)
+
+	// Remove existing pointclouds, but leave maps and config (so we keep the lua files).
+	test.That(t, resetFolder(name+"/data"), test.ShouldBeNil)
+	// Count the initial number of maps in the map directory
+	numMaps, err := ioutil.ReadDir(name + "/map/")
+	test.That(t, err, test.ShouldBeNil)
+	// Test online mode using the map generated in the offline test
```

Comment:
[opt] I thinking the log line is selfexplanatory

Reply:
I do actually like these comments. They make it easier to read the code for me.

Reply:
had the comment because there was one initially for the offline mode test